ipaclient/install/ipadiscovery.py

```diff
@@ -521,7 +530,16 @@ def ipadnssearchkrbrealm(self, domain=None):
         root_logger.debug("Search DNS for TXT record of %s", qname)
 
         try:
-            answers = resolver.query(qname, rdatatype.TXT)
+            answers = []
+            dns_answers = resolver.query(qname, rdatatype.SRV)
```

Comment:
You changed `TXT` to `SRV` here. It should stay TXT RR and it has no priority attribute